README.md

```diff
@@ -15,7 +24,11 @@ City of Helsinki ground rent system
 3. Create superuser if needed:
    - `docker exec -it mvj python manage.py createsuperuser`
 
-The project is now running at [localhost:8000](http://localhost:8000)
+The project is now running at [localhost:8000](http://localhost:8000).
+
+Known issues:
+- runserver_plus not found/not working: replace `command: python manage.py runserver_plus 0:8000` with `command: python manage.py runserver 0:8000` command in `docker-compose.yml`.
```

Comment:
For me this was fixed by installing `django-extensions` separately. It is weird since it is listed in requirements

Reply:
As discussed in meeting, this was probably some glitch, because not everyone had to install `django-extensions` manually.